src/commands/edge/channels/assign.ts

```diff
@@ -0,0 +1,64 @@
+import { type ArgumentsCamelCase, type Argv, type CommandModule } from 'yargs'
+
+import { apiCommand, apiCommandBuilder, apiDocsURL, type APICommandFlags } from '../../../lib/command/api-command.js'
+
+import { chooseChannel } from '../../../lib/command/util/edge/channels-choose.js'
+import { chooseDriver } from '../../../lib/command/util/drivers-choose.js'
+
+
+export type CommandArgs =
+	& APICommandFlags
+	& {
+		driverId?: string
+		driverVersion?: string
+		channel?: string
+	}
+
+const command = 'edge:channels:assign [driver-id] [driver-version]'
+
+const describe = 'assign a driver to a channel'
+
+const builder = (yargs: Argv): Argv<CommandArgs> =>
+	apiCommandBuilder(yargs)
+		.positional('driver-id', { describe: 'driver id', type: 'string' })
+		.positional('driver-version', { describe: 'driver version', type: 'string' })
+		.option('channel', { alias: 'C', describe: 'channel to assigned to', type: 'string' })
+		.example([
+			['$0 edge:channels:assign', 'prompt for a channel and driver'],
+			[
+				'$0 edge:channels:assign 636169e4-8b9f-4438-a941-953b0d617231',
+				'prompt for a channel and assign the specified driver to it',
+			],
+			[
+				'$0 edge:channels:assign --channel 6ea857c6-23d4-4aae-aaf7-7a36daf42f92',
+				'prompt for a driver and assign it to the specified channel',
+			],
+			[
+				'$0 edge:channels:assign 636169e4-8b9f-4438-a941-953b0d617231' +
+					' --channel 6ea857c6-23d4-4aae-aaf7-7a36daf42f92',
+				'assign the specified driver to the specified channel',
+			],
+		])
+		.epilog(apiDocsURL('createDriverChannel'))
```

Comment:
Should there also be an example using `driver-version`?

Reply:
I didn't include one because that is rarely, if ever, needed.